classes/Configuration.php

```diff
@@ -255,7 +255,7 @@ public static function getGlobalValue($key, $idLang = null)
     }
 
     /**
-     * @deprecated use Configuration::getConfigInMultipleLangs() instead.
+     * @deprecated use Configuration::getConfigInMultipleLangs() instead
```

Comment:
is there a known reason why cs fixer doesn't allow a dot at the end of comment?

Reply:
See `phpdoc_annotation_without_dot` https://github.com/FriendsOfPHP/PHP-CS-Fixer
"PHPDoc annotation descriptions should not be a sentence."

Dont know why but it's documented